posts/2023-09-20-Primeur.adoc

```diff
@@ -0,0 +1,53 @@
+
+
+== Why choose Open Liberty?
+
+Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem contradictory and might not be something you have considered when selecting a cloud-native runtime, they are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty to run its business application, whether on-prem or in a cloud environment.
+
+image::img/blog/primeur_logo.png[Primeur Logo ,width=70%,align="center"]
+
+== Invisibility
+
+The great benefit of Open Liberty’s ability to remain invisible is that Primeur can embed, completely configure, and operate Open Liberty within their DATA ONE product without their customers knowing that Open Liberty is even there. Their customers need only care about DATA ONE. So customers’ developers don’t have to skill up on yet another technology or care about how to operate the runtime that their platform is based upon.
+
+
+== Observability
+
+In almost complete contrast to invisibility, Primeur also required a runtime that enables great observability. As an ISV, shipping products, installing them at customer sites in the most frictionless way, and adopting them for production environments is just the beginning. When customers raise problems, the complexity of cloud-native application architectures and distributed cloud infrastructure can make it difficult to detect bottlenecks and failure points. Primeur's engineers can effectively and efficiently use Open Liberty's problem-determination tools to monitor, trace, and generally probe Open Liberty until they can identify the root cause of any problems reported by their customers. Moreover, Primeur's combined use of Open Liberty and link:https://developer.ibm.com/blogs/introducing-the-ibm-semeru-runtimes/[IBM Semeru Runtimes JDK] enables them to also easily diagnose problems at the JVM level.
+
+
+
+
+== Additional valuable Liberty features
+
+Primeur also liked the flexibility of Open Liberty in helping their customers to move at their own pace from more traditional deployment models to containerized deployments in the cloud. Open Liberty supports the use of cloud-native frameworks and platforms, like Jakarta EE, MicroProfile, and OpenShift. Open Liberty also enables Primeur to deploy DATA ONE either natively or in containers, according to their customers’ preferences and skills. This flexibility means that their customers can transition from traditional to containerized deployments at their own pace.
+
+Open Liberty’s frequent updates with security and non-security fixes, and its zero migration architecture, mean that Primeur’s DATA ONE can be kept up-to-date and secure. They can also access IBM’s support to mitigate complex risks and to solve challenging problems.
```

Comment:
```suggestion
Open Liberty’s frequent updates with security and non-security fixes, and its zero migration architecture, mean that Primeur’s DATA ONE can be kept up-to-date and secure. It can also access IBM’s support to mitigate complex risks and to solve challenging problems.
```